meson.build

```diff
@@ -283,13 +283,75 @@ io_long_double = get_option('io-long-double')
 # Select stdio implementation
 tinystdio = get_option('tinystdio')
 
-has_link_defsym = meson.get_cross_property('has_link_defsym',
-                                           cc.has_link_argument('-Wl,--defsym=' + 'start=0') or
-                                           cc.has_link_argument('-Wl,--defsym=' + '_start=0') or
-                                           cc.has_link_argument('-Wl,--defsym=' + '__start=0') or
-                                           cc.has_link_argument('-Wl,--defsym=' + '___start=0') )
-has_link_alias = meson.get_cross_property('has_link_alias', cc.has_link_argument('-Wl,-alias,' + global_prefix + 'main,testalias'))
-lib_gcc = meson.get_cross_property('libgcc', '-lgcc')
+# Figure out what flags the linker supports
+
+has_link_defsym = false
+has_link_alias = false
+
+
+# Figure out how to pass aliases on the compiler command line.  Do
+# that by finding a suitable alias for the default entry point.
+
+foreach start : ['start', '_start', '__start', '___start']
```

Comment:
Would using -Wl,-e,main be simpler to override the default entry point?

Reply:
Possibly, but we need to know how to create aliases anyways for vfprintf and vfscanf, and I'm not sure how to handle linkers that don't support -e.

Reply:
Fair, if this current approach works that sounds good to me

Reply:
thanks for your review!